benchmark/libponyc/CMakeLists.txt

```diff
@@ -26,19 +24,17 @@ target_link_libraries(libponyc.benchmarks
     PRIVATE benchmark::benchmark
 )
 
+target_link_libraries(libponyc.benchmarks PRIVATE ${PONYC_LLVM_LIBS})
+
 if (MSVC)
     target_link_libraries(libponyc.benchmarks
         PRIVATE dbghelp
         PRIVATE ucrt
         PRIVATE Ws2_32
         PRIVATE Shlwapi
     )
-
-    get_target_property(TARGET_LIBRARIES libponyc.benchmarks LINK_LIBRARIES)
-    LIST(REMOVE_ITEM TARGET_LIBRARIES LTO)
-    set_property(TARGET libponyc.benchmarks PROPERTY LINK_LIBRARIES ${TARGET_LIBRARIES})
 elseif(${CMAKE_HOST_SYSTEM_NAME} MATCHES "Darwin")
-    target_link_libraries(libponyc.benchmarks PRIVATE dl)
+    target_link_libraries(libponyc.benchmarks PRIVATE dl z)
```

Comment:
I spent a lot of time getting rid of the `z` dependency when I was getting the standalone libponyc working. Do we really need it- there wasn't any reason for it previously.

Reply:
I think that snuck in when I merged the M1 branch; I'll remove it.